debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java

```diff
@@ -93,7 +93,7 @@ protected String zookeeperConnection() {
      */
     protected void populateDefaultConfiguration(Properties props) {
         config.setProperty(KafkaConfig.NumPartitionsProp(), String.valueOf(1));
-        config.setProperty(KafkaConfig.LogFlushIntervalMessagesProp(), String.valueOf(1));
+        config.setProperty(KafkaConfig.LogFlushIntervalMessagesProp(), String.valueOf(Long.MAX_VALUE));
```

Comment:
Do we even need to set it at all if that's just the default?

Reply:
Good question, I inteded to keep it there just in case it had *some* reason in the past. It might help in future if it turns out it is important for something. If we remove it this mental link will get lost.
But both works for me.